src/gui/qgslayermetadatasearchwidget.cpp

```diff
@@ -48,6 +48,19 @@ QgsLayerMetadataSearchWidget::QgsLayerMetadataSearchWidget( QWidget *parent, Qt:
   mExtentFilterComboBox->addItem( QStringLiteral( "Map Canvas Extent" ) );
   mExtentFilterComboBox->addItem( QStringLiteral( "Current Project Extent" ) );
   mExtentFilterComboBox->setCurrentIndex( 0 );
+  mExtentFilterComboBox->setSizeAdjustPolicy( QComboBox::SizeAdjustPolicy::AdjustToContents );
+  mExtentFilterComboBox->adjustSize();
+
+  mGeometryTypeComboBox->addItem( QString( ), QVariant() );
```

Comment:
Can you add icons to this combo too? It makes a big usability difference

Reply:
done!